SU2_CFD/src/output/CFlowCompOutput.cpp

```diff
@@ -726,7 +726,7 @@ bool CFlowCompOutput::SetInit_Residuals(CConfig *config){
 
 bool CFlowCompOutput::SetUpdate_Averages(CConfig *config){
 
-  return true;
+  return (config->GetTime_Marching() != TIME_MARCHING::STEADY && (curInnerIter == config->GetnInner_Iter() - 1 || convergence));
```

Comment:
It is strange that would break the testcase, I'll have a look.

Reply:
Your initial change was correct, the logic was artificially delaying the update of the average to the end of the timestep, and the timedomain regressions check the first inner iteration by default.